packages/flame_forge2d/lib/body_component.dart

```diff
@@ -133,4 +133,10 @@ abstract class BodyComponent<T extends Forge2DGame> extends BaseComponent
   bool containsPoint(Vector2 point) {
     return body.fixtures.any((fixture) => fixture.testPoint(point));
   }
+
+  @override
+  void onRemove() {
```

Comment:
This seems to be a different fix right? It deserves its own PR, or its own entry on the changelog

Reply:
it's not a fix, just changing the way it's done to work well with this PR

Reply:
Still, at least a changelog entry is needed right, this is on another package.

Reply:
I don't see why, there is no functional difference right? just impl detail

Reply:
but I can add one